src/nodes/concat.h

```diff
@@ -0,0 +1,114 @@
+
+namespace toC {
+
+    class Concat : public Node {
+    public:
+        Concat() {
+            op_name = "Concat";
+            output = nullptr;
+        }
+
+        int axis = 1;
+        std::vector<const Tensor *> node_inputs;
+        const Tensor *output;
+
+        void print_parameters(std::ostream &dst, bool decorate) const override {
+            size_t input_count = node_inputs.size();
+            for (size_t i = 0; i < input_count; i++) {
+                node_inputs[i]->print_tensor(dst, !decorate);
+                if (i < (input_count - 1))
+                    dst << ", ";
+            }
```

Comment:
The output tensor needs to be printed

Reply:
Indeed! I missed that one - doesn't interfere with the backend tests. I will fix it asap!

Reply:
Yup. I would even say not catching this is a bug in the backend test generator...